Permanent/ViewControllers/Authentication/LoginViewController.swift

```diff
@@ -107,7 +107,18 @@ class LoginViewController: BaseViewController<AuthViewModel> {
             showAlert(title: .success,
                       message: String(format: .emailSent, email!))
         case .error(let message):
-            showAlert(title: .error, message: message)
+            DispatchQueue.main.async {
+                let alert = UIAlertController(title: .error, message: message, preferredStyle: .alert)
+                
+                alert.addAction(UIAlertAction(title: .cancel, style: .cancel, handler: nil))
+                alert.addAction(UIAlertAction(title: .retry, style: .default, handler: {_ in
+                    self.dismiss(animated: true, completion: {
```

Comment:
Theoretically the action is executed after dismissal of the alert. can you please double check?

Reply:
Checked and there is no need for dismiss.